build-doc.sh

```diff
@@ -127,7 +127,7 @@ num_cpu()
                         ncpu=$(grep -c "^processor" /proc/cpuinfo)
                 elif [ -x /sbin/sysctl ]; then
                         ncpu=$(/sbin/sysctl -n hw.ncpu 2>/dev/null)
-                        if [ $? -neq 0 ]; then
```

Comment:
There is no such thing as an operator `-neq`.

Reply:
Indeed !